nalgebra-glm/src/common.rs

```diff
@@ -133,6 +133,8 @@ pub fn clamp_vec<T: Number, const D: usize>(
 ///
 /// The floating-point value's bit-level representation is preserved.
 ///
+/// Using unsafe is sound because the bitwise representation of f32 fits in i32
```

Comment:
This documentation goes inside the function, on the `unsafe` block. The use of unsafety in the implementation is not public; the purpose of the comment is to help a developer or auditor who is looking at the implementation.